.env

```diff
@@ -0,0 +1,5 @@
+ALGOLIA_APP_ID={{ YOUR_APP_ID }}
+SEARCH_ONLY_API_KEY={{ SEARCH_ONLY_API_KEY }}
+ALGOLIA_ADMIN_KEY={{ YOUR_ADMIN_KEY }}
+ALGOLIA_INDEX_NAME={{ YOUR_INDEX_NAME }}
```

Comment:
I assume we'd need to add this to the build, not sure if the `$YOUR_...` bits are needed, we can simply ensure that the correct secrets are loaded to the environmet under `$ALGOLIA...`. Can you comment on which of these are secret and which are public? I'm thinking that the `ALGOLIA_INDEX_NAME` might be public.

Reply:
Hello Mr.Zoran I think APP_ID and INDEX_NAME can be public ill put exacts values into them, I`m not sure whether SEARCH_ONLY_API_KEY public or not but ADMIN_KEY is definitely not public.